CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Add metrics on updating organization data
```

Comment:
can we specify these are metrics about adding/updating/removing an user from an organization?

Reply:
I was waiting the merge of another PR to put the changelog info and not have conflicts.